easybuild/easyconfigs/f/fmt/fmt-8.1.1-GCCcore-11.2.0.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'CMakeMake'
+
+name = 'fmt'
+version = '8.1.1'
+
+homepage = 'http://fmtlib.net/'
+description = "fmt (formerly cppformat) is an open-source formatting library."
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
```

not needed when configuring with `-DCMAKE_POSTITION_INDEPENDENT_CODE=TRUE`
```suggestion
toolchainopts = {'pic': True}
```

---

As above. What is the 'better' way of doing it? I am sure something has slipped my attention here.